src/algorithms/LaurentPoly.jl

```diff
@@ -88,7 +88,7 @@ end
 # return the degree of the unique term, throw if not a term
 function term_degree(p::LaurentPolyRingElem)
    isnull = true
-   local deg
+   local deg = -1
```

Comment:
```suggestion
   deg = -1
```
does one still need the `local`?

Reply:
I think not (I admit I simply made minimal changes. And that over a month ago, I had forgotten about this incomplete PR...)